### PR TITLE
mqbcfg.xsd: Increase failover timeout to 5 minutes for proxies

### PR DESCRIPTION
**Pull request by kaikulimu**
Looking through the code, I realize that proxies does not use the cluster definitions from [`clusters.json`](https://bbgithub.dev.bloomberg.com/BMQ/bmqdomains/blob/main/clusters.json) at all. If you look at the constructor of [`ClusterProxy`](https://github.com/bloomberg/blazingmq/blob/b822147d4a5bfe80fe12139d585876ccc93b6b23/src/groups/mqb/mqbblp/mqbblp_clusterproxy.cpp#L1023-L1035), it is passed a default-initialized `mqbcfg::ClusterDefinition(allocator)`. Therefore, it will only use the default values, which is [2 minutes](https://github.com/bloomberg/blazingmq/blob/b822147d4a5bfe80fe12139d585876ccc93b6b23/src/groups/mqb/mqbcfg/mqbcfg.xsd#L622) in the case of failover threshold. Therefore, I am raising the default to 5 minutes to make all proxies have a failover timeout of 5 minutes.

Clusters nodes obey definitions in `clusters.json`, so they should not be affected by this change.

### CODE REVIEW

File: src/groups/mqb/mqbcfg/mqbcfg_messages.cpp [hunk 1]
```diff
@@ -335,15 +272,15 @@ const int ClusterMonitorConfig::DEFAULT_INITIALIZER_MAX_TIME_MASTER = 120;
 
 const int ClusterMonitorConfig::DEFAULT_INITIALIZER_MAX_TIME_NODE = 120;
 
-const int ClusterMonitorConfig::DEFAULT_INITIALIZER_MAX_TIME_FAILOVER = 240;
```
Comment (pniedzielski) [inline src/groups/mqb/mqbcfg/mqbcfg_messages.cpp:338]: This is the meat and potatoes.  Everything else is noise.

File: src/groups/mqb/mqbcfg/mqbcfg_messages.cpp [hunk 2]
```diff
@@ -6330,12 +5804,12 @@ Configuration::print(bsl::ostream& stream, int level, int spacesPerLevel) const
 }  // close package namespace
 }  // close enterprise namespace
 
-// GENERATED BY BLP_BAS_CODEGEN_2023.10.07
+// GENERATED BY @BLP_BAS_CODEGEN_VERSION@
```
Comment (pniedzielski) [inline src/groups/mqb/mqbcfg/mqbcfg_messages.cpp:5807]: This looks suspicious.

**Review by pniedzielski — APPROVED** [review]
Change to the XSD file looks good.

I was a little confused by the diffs in `mqbcfg_messages`, but it just looks like the codegen changed where it wants to put definitions/changed to inlining more functions.  That's okay with me.  CI passes.  I just see one place where a version string isn't being outputted; I'm not sure if this is important or not.

File: src/groups/mqb/mqbcfg/mqbcfg_messages.cpp (hunk 2 above)
Comment (kaikulimu) [reply src/groups/mqb/mqbcfg/mqbcfg_messages.cpp:5807]: I am using the newest version of BAS codegen. For some reason, this string substitution doesn't work but shouldn't be a big deal.

**Pull request merged**